.github/workflows/release-go.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Release
+
+# on:
```

Comment:
shouldn't this run on a schedule?

Reply:
Yes, it should, it is commented out to avoid accidental releasing